stl/inc/yvals_core.h

```diff
@@ -1778,7 +1779,10 @@ _EMIT_STL_ERROR(STL1004, "C++98 unexpected() is incompatible with C++23 unexpect
 #endif // _HAS_CXX20
 
 // C++23
-#if _HAS_CXX23
+#if _HAS_CXX23
+#ifdef __clang__ // Clang only for now
+#define __cpp_lib_is_implicit_lifetime 202302L
```

Comment:
You need to add test for this macro, to this file: [`VSO_0157762_feature_test_macros/test.compile.pass.cpp`](https://github.com/microsoft/STL/blob/main/tests/std/tests/VSO_0157762_feature_test_macros/test.compile.pass.cpp).

Note that the macro names are ordered alphabetically in the test file.